web/Dockerfile

```diff
@@ -12,6 +12,7 @@ FROM node:alpine AS builder
 WORKDIR /app
 COPY . .
 COPY --from=deps /app/node_modules ./node_modules
+ENV NODE_OPTIONS --openssl-legacy-provider
```

Comment:
Is it worth fixing the alpine version for which this works?

Reply:
As you said, fixing the version seems to be a better way to prevent this kind of break. 👍 
I close this PR because another PR #45 was just created and will fix this issue by fixing the version. Thanks for reviewing.